### DISCUSSION TITLE
How to configure WCA API with wordpress instance?

### DISCUSSION THREAD

**Discussion by farhanulju** [thewca/worldcubeassociation.org#6906] [Q&A]
I am trying to integrate the API with a WordPress instance. 

Purpose:
WCA login in website to get their profile and Name

Using this plugin: https://wordpress.org/plugins/miniorange-login-with-eve-online-google-facebook/

I have configured it like this. But I am getting errors. 

![Screenshot 2022-05-16 023525](https://user-images.githubusercontent.com/72323749/168492873-1b42fe14-f71c-4fc3-9de2-895c755538c1.jpg)
![Screenshot 2022-05-16 023600](https://user-images.githubusercontent.com/72323749/168492870-43d11159-7e37-4ef2-81a8-d039b5fa47ed.jpg)
I'm guessing the API here is Oauth 2.0. Correct me if I am wrong.

**Comment by gregorbg** [comment]
Hey! We don't currently have the capacity to provide user support for our OAuth endpoints. So unfortunately, I cannot look into your problem in great detail. The only thing that I can tell you is there are a lot of implementations out there which work fine, so you can take existing code as an example.

You could also refer to our Wiki at https://github.com/thewca/worldcubeassociation.org/wiki/OAuth-documentation-notes and a reference implementation at https://github.com/thewca/wca-oauth

Happy coding!
- Gregor

**Reply by itsmemidhuz123** [reply]
its a great thing that wca could provide an opportunity for this, so our shopping users could signin using wca account as well

**Reply by farhanulju** [reply]
Same goes for the user profile event details and timings.

Currently, we need to download the whole WCA database and process it locally to showcase the user profile data for each event.
Having API for that would have been much simpler SSO integration.

But this would obviously impact WCA API requests server. So I can see why it may never see the light of production.

**Reply by itsmemidhuz123** [reply]
Yeah that's corect
